doc/source/users/environment_variable_catalog.rst

```diff
@@ -112,6 +112,19 @@ LogMacroFormat
 
 Definition of the format to be used to save the file.
 
+.. _logmacrofilter:
+
+LogMacroFilter
```

Comment:
This is not an environment variable but sardana custom setting. I would put this documentation as a docstring sardanacustomsettings module.

Reply:
You can reference a sardana custom setting with [data Python domain markup](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py:data) (see [example](https://sardana-controls.org/users/configuration/spock.html))

Reply:
Sorry, it is true ... I implemented and I forgot about it !